From c0eb88dc0bef345a9c8593fa77dd98b7f8cfb1cb Sat, 23 May 2020 10:09:50 +0200
From: Andrew Branson <andrew.branson@jolla.com>
Date: Sat, 23 May 2020 10:09:16 +0200
Subject: [PATCH] Add tracker-sparql-2.0 to the list to check

diff --git a/configure.ac b/configure.ac
index 285a297..a8f25d4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -167,7 +167,7 @@
 
 # Search available Tracker version
 trackerpkg="tracker-sparql-0.10"
-m4_foreach([VERSION], [[0.10], [0.11], [0.12], [0.14], [0.15], [0.16], [0.17], [0.18], [1.0]],
+m4_foreach([VERSION], [[0.10], [0.11], [0.12], [0.14], [0.15], [0.16], [0.17], [0.18], [1.0], [2.0]],
                       [PKG_CHECK_EXISTS([tracker-sparql-VERSION >= 0.10.5],
                                         [ trackerpkg="tracker-sparql-VERSION" ])
                       ])
